test/inductor/test_cudagraph_trees.py

```diff
@@ -50,6 +52,13 @@
 from io import StringIO
 
 
+def get_compile_fn(backend):
+    if backend == "cudagraphs":
+        return functools.partial(torch.compile, backend="cudagraphs")
```

Comment:
Don't you also have an option to do inductor + cudagraphs (but not cudagraph trees?) or am I hallucinating

Reply:
That's basically old, legacy code. kept around because occasionally useful for debugging but not actively used